dbt/models/default/schema/default.pinval_comp.yml

```diff
@@ -0,0 +1,14 @@
+models:
+  - name: default.pinval_comp
+    description: '{{ doc("pinval_comp") }}'
+    data_tests:
+      - unique_combination_of_columns:
+          name: default_pinval_comp_unique_run_id_pin_card_comp_num
+          combination_of_columns:
+            - run_id
+            - pin
+            - card
+            - comp_num
+          allowed_duplicates: 0
+          config:
+            error_if: ">1"
```

Comment:
This [test fails](https://github.com/ccao-data/data-architecture/actions/runs/14580754916/job/40896744099), but I'm not sure these  [these](https://github.com/ccao-data/pinval/issues/34) should be the unique keys. If this were to hold, then the same pin/card combo could not be used twice as a comp for different properties, which seems like it could be a relatively common occurrence